data/author/mathew-korica.json

```diff
@@ -0,0 +1,6 @@
+{
+   "name" : "Mathew Korica",
+   "bio" : "",
```

Comment:
Could you please fill some `bio`?

Reply:
You can leave that blank (as is the case with other authors).

Reply:
OK, we can do that